OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -331,9 +331,12 @@ bool IsBlockedBy(Actor self, Actor otherActor, Actor ignoreActor, BlockedByActor
 
 			// If the check allows: We are not blocked by units that we can force to move out of the way.
 			if (check <= BlockedByActor.Immovable && cellFlag.HasCellFlag(CellFlag.HasMovableActor) &&
-				self.Owner.Stances[otherActor.Owner] == Stance.Ally &&
-				otherActor.TraitOrDefault<IMove>() != null)
-				return false;
+				self.Owner.Stances[otherActor.Owner] == Stance.Ally)
+			{
+				var mobile = otherActor.TraitOrDefault<Mobile>();
+				if (mobile != null && !mobile.IsTraitDisabled && !mobile.IsTraitPaused & !mobile.RequireForceMove)
```

Comment:
I suppose that single & wants to be an &&?

Reply:
Fixed.